processing file asmFiles/beq_not_taken.asm (1 of 23)
processing file asmFiles/beq_taken.asm (2 of 23)
processing file asmFiles/bne_not_taken.asm (3 of 23)
processing file asmFiles/bne_taken.asm (4 of 23)
processing file asmFiles/branch_hazard.asm (5 of 23)
processing file asmFiles/count_days.asm (6 of 23)
--- memsim.hex	2021-10-10 03:28:00.968114676 -0400
+++ memcpu.hex	2021-10-10 03:28:00.892111811 -0400
@@ -63,7 +63,7 @@
 :04003E00AFA6000069
 :04003F0003E00008D2
 :043FFB000000001EA4
-:043FFC00000000D2EF
-:043FFD0000000E4270
-:043FFE0000000F327E
+:043FFC000000002A97
+:043FFD00000001AE11
+:043FFE00000001F6C8
 :00000001FF
processing file asmFiles/fib.asm (7 of 23)
processing file asmFiles/jal.asm (8 of 23)
processing file asmFiles/jump_hazard.asm (9 of 23)
processing file asmFiles/load_use.asm (10 of 23)
processing file asmFiles/mergesort.asm (11 of 23)
processing file asmFiles/mult.asm (12 of 23)
processing file asmFiles/mult_procedure.asm (13 of 23)
--- memsim.hex	2021-10-10 03:28:29.231180301 -0400
+++ memcpu.hex	2021-10-10 03:28:29.151177285 -0400
@@ -32,5 +32,5 @@
 :04001F0003E00008F2
 :043FFC0000000010B1
 :043FFD00000000A020
-:043FFE00000013604C
+:043FFE0000000BA014
 :00000001FF
processing file asmFiles/RAW.asm (14 of 23)
processing file asmFiles/search.asm (15 of 23)
processing file asmFiles/test.branchjump1.asm (16 of 23)
processing file asmFiles/test_halt.asm (17 of 23)
processing file asmFiles/test_jal_jr.asm (18 of 23)
processing file asmFiles/test_j.asm (19 of 23)
processing file asmFiles/test.ldst1.asm (20 of 23)
processing file asmFiles/test.ldst2.asm (21 of 23)
processing file asmFiles/test.loadstore.asm (22 of 23)
processing file asmFiles/test.rtype.asm (23 of 23)
processing file asmFiles/beq_not_taken.asm (1 of 23)
processing file asmFiles/beq_taken.asm (2 of 23)
processing file asmFiles/bne_not_taken.asm (3 of 23)
processing file asmFiles/bne_taken.asm (4 of 23)
processing file asmFiles/branch_hazard.asm (5 of 23)
processing file asmFiles/count_days.asm (6 of 23)
--- memsim.hex	2021-10-10 04:37:53.461149402 -0400
+++ memcpu.hex	2021-10-10 04:37:53.356145443 -0400
@@ -63,7 +63,7 @@
 :04003E00AFA6000069
 :04003F0003E00008D2
 :043FFB000000001EA4
-:043FFC00000000D2EF
-:043FFD0000000E4270
-:043FFE0000000F327E
+:043FFC000000002A97
+:043FFD00000001AE11
+:043FFE00000001F6C8
 :00000001FF
processing file asmFiles/fib.asm (7 of 23)
processing file asmFiles/jal.asm (8 of 23)
processing file asmFiles/jump_hazard.asm (9 of 23)
processing file asmFiles/load_use.asm (10 of 23)
processing file asmFiles/mergesort.asm (11 of 23)
processing file asmFiles/mult.asm (12 of 23)
processing file asmFiles/mult_procedure.asm (13 of 23)
--- memsim.hex	2021-10-10 04:38:21.859219983 -0400
+++ memcpu.hex	2021-10-10 04:38:21.788217307 -0400
@@ -32,5 +32,5 @@
 :04001F0003E00008F2
 :043FFC0000000010B1
 :043FFD00000000A020
-:043FFE00000013604C
+:043FFE0000000BA014
 :00000001FF
processing file asmFiles/RAW.asm (14 of 23)
processing file asmFiles/search.asm (15 of 23)
processing file asmFiles/test.branchjump1.asm (16 of 23)
processing file asmFiles/test_halt.asm (17 of 23)
processing file asmFiles/test_jal_jr.asm (18 of 23)
processing file asmFiles/test_j.asm (19 of 23)
processing file asmFiles/test.ldst1.asm (20 of 23)
processing file asmFiles/test.ldst2.asm (21 of 23)
processing file asmFiles/test.loadstore.asm (22 of 23)
processing file asmFiles/test.rtype.asm (23 of 23)
processing file asmFiles/beq_not_taken.asm (1 of 23)
processing file asmFiles/beq_taken.asm (2 of 23)
--- memsim.hex	2021-10-10 04:39:58.556865393 -0400
+++ memcpu.hex	2021-10-10 04:39:58.473862264 -0400
@@ -9,5 +9,4 @@
 :04000800ACA200089E
 :040009001044FFFDA3
 :04000A001062FFFA87
-:043FC20000000009F2
 :00000001FF
processing file asmFiles/bne_not_taken.asm (3 of 23)
processing file asmFiles/bne_taken.asm (4 of 23)
--- memsim.hex	2021-10-10 04:40:04.941106073 -0400
+++ memcpu.hex	2021-10-10 04:40:04.858102944 -0400
@@ -8,5 +8,4 @@
 :04000700AC820008BF
 :040008001482FFFB64
 :04000900FFFFFFFFF7
-:043FC20000000009F2
 :00000001FF
processing file asmFiles/branch_hazard.asm (5 of 23)
--- memsim.hex	2021-10-10 04:40:08.242230522 -0400
+++ memcpu.hex	2021-10-10 04:40:08.171227845 -0400
@@ -7,5 +7,5 @@
 :04000600FFFFFFFFFA
 :04000700AC25000024
 :04000800FFFFFFFFF8
-:04003C0000000002BE
+:04003C0000000001BF
 :00000001FF
processing file asmFiles/count_days.asm (6 of 23)
--- memsim.hex	2021-10-10 04:40:11.231343209 -0400
+++ memcpu.hex	2021-10-10 04:40:11.167340797 -0400
@@ -62,8 +62,9 @@
 :04003D0027BDFFFCE0
 :04003E00AFA6000069
 :04003F0003E00008D2
-:043FFB000000001EA4
-:043FFC00000000D2EF
-:043FFD0000000E4270
-:043FFE0000000F327E
+:043FFA000000001EA5
+:043FFB0000000009B9
+:043FFC00000001942C
+:043FFD0000000008B8
+:043FFE000000001EA1
 :00000001FF
processing file asmFiles/fib.asm (7 of 23)
processing file asmFiles/beq_not_taken.asm (1 of 23)
processing file asmFiles/beq_taken.asm (2 of 23)
processing file asmFiles/bne_not_taken.asm (3 of 23)
processing file asmFiles/bne_taken.asm (4 of 23)
processing file asmFiles/branch_hazard.asm (5 of 23)
processing file asmFiles/count_days.asm (6 of 23)
--- memsim.hex	2021-10-10 04:42:17.344097541 -0400
+++ memcpu.hex	2021-10-10 04:42:17.278095053 -0400
@@ -63,7 +63,7 @@
 :04003E00AFA6000069
 :04003F0003E00008D2
 :043FFB000000001EA4
-:043FFC00000000D2EF
-:043FFD0000000E4270
-:043FFE0000000F327E
+:043FFC000000002A97
+:043FFD00000001AE11
+:043FFE00000001F6C8
 :00000001FF
processing file asmFiles/fib.asm (7 of 23)
processing file asmFiles/jal.asm (8 of 23)
